edweiss-app/components/input/FancyTextInput.tsx

```diff
@@ -23,9 +23,10 @@ type FancyTextInputProps = MarginProps & Testable & {
 	readOnly?: boolean;
 	label?: string;
 	error?: string;
+	numeric?: boolean;
 };
 
-const FancyTextInput: ReactComponent<FancyTextInputProps> = ({ backgroundColor = 'crust', textColor = 'text', placeholderColor = 'overlay0', multiline, ...props }) => {
+const FancyTextInput: ReactComponent<FancyTextInputProps> = ({ backgroundColor = 'crust', textColor = 'text', placeholderColor = 'overlay0', multiline, numeric, ...props }) => {
```

Comment:
Can you please explain me what "numeric" does?

Reply:
That's a great question!
Priorly the `FancyTextInput` used to insert text allowed us to enter random string. In my case a page is a non negative number. So I have added this possibility in order to enter only number (as a calculator, so to say). It's a new boolean called `numeric`, that when enabled will show a numeric keyboard.

Does it make more sens ? What do you think about this small change ?

Reply:
Ah, I see now—thanks for the explanation! That makes perfect sense, especially since a page should only ever be a non-negative number. I think it's a great addition to enforce that constraint directly in the input rather than handling it later. The numeric keyboard is a nice touch too for user convenience and I am sure it would be very useful for other modules too!

As for the change, it seems small but meaningful—it helps maintain data integrity and improves UX. Nice work on this!